meta-pike/recipes-android/android/android_pike-o.bb

```diff
@@ -0,0 +1,41 @@
+inherit gettext
+
+SUMMARY = "Downloads the Polar M600 /system and /usr/include/android folders and installs them for libhybris"
+LICENSE = "CLOSED"
+SRC_URI = "https://dl.dropboxusercontent.com/s/8b9t2renrxbl4gq/hybris-o-msm8909.tar.gz;name=hybris"
```

Comment:
This is the same one used for other Oreo bases right? Maybe it would make sense to add a 'general' recipe for it at https://github.com/AsteroidOS/meta-asteroid/tree/master/recipes-android/android?

Obviously this could be done in another PR.

Reply:
this is probably work that will go in meta-asteroid (similar to the msm8909 platforms for pie) so I'm not going to do anything here for now